src/cli/main.c

```diff
@@ -32,6 +32,7 @@ static struct argp usage = {
                 {"user", 'u', "UID[:GID]", OPTION_ARG_OPTIONAL, "User and group to use for privilege separation", -1},
                 {"root", 'r', "PATH", 0, "Path to the driver root directory", -1},
                 {"ldcache", 'l', "FILE", 0, "Path to the system's DSO cache", -1},
+                {"no-create-imex-channels", 0x80, NULL, 0, "Skip the creation of IMEX channel device nodes", -1},
```

Comment:
Concerning the name of the flag, is there precedent for naming the option as `no-*`? What about `disable-imex-channel-creation`?

Reply:
The `no-*` prefix is used for other negative options across this project:
```
        {"no-glvnd", OPT_NO_GLVND},
        {"no-uvm", OPT_NO_UVM},
        {"no-modeset", OPT_NO_MODESET},
        {"no-mps", OPT_NO_MPS},
```
and
```
                {"no-cgroups", 0x84, NULL, 0, "Don't use cgroup enforcement", -1},
                {"no-devbind", 0x85, NULL, 0, "Don't bind mount devices", -1},
```
In addition to more recently added options. I'm happy to update, but was trying to be locally consistent.

Reply:
+1 for Chris' comment--if we can avoid "no" prefixes for new projects that'll be a win.  I want to think that I noticed the precedent and refrained from commenting.

Thanks Evan for doing more digging.  While I prefer "enable" and "disable", based on the precedent of "Don't <blah>...", I'd like to revise my comment to a request for "Don't automatically create IMEX channel device nodes".

Reply:
Thanks for the context. I am okay to keep the naming as-is considering the large precedent.

Reply:
I have updated the usage string. Is there a more resonable `no-`-prefixed string that we could use here? Otherwise I will stick with what we currently have. 

@mikemckiernan I definitely agree on avoiding this in newer projects!

Reply:
LGTM